birdhouse/components/cowbird/config/cowbird/config.yml.template

```diff
@@ -0,0 +1,155 @@
+## Combined configuration definition to setup objects at Cowbird startup
+## See documentation 'Configuration' for more details on each section.
+
+## See example: https://github.com/Ouranosinc/cowbird/blob/master/config/config.example.yml
+## See details: https://pavics-cowbird.readthedocs.io/en/latest/configuration.html#file-config-yml
+
+
+# [Required] Definition of managed services by Cowbird.
+# FIXME: rename 'services' section to 'handlers' and refactor (issue https://github.com/Ouranosinc/cowbird/issues/22)
+services:
+  Magpie:
+    active: true
+    url: https://${PAVICS_FQDN_PUBLIC}/magpie
+    admin_user: ${MAGPIE_ADMIN_USERNAME}
+    admin_password: ${MAGPIE_ADMIN_PASSWORD}
+  Geoserver:
+    active: true
+    url: http://${PAVICS_FQDN_PUBLIC}:8765/geoserver
+    workspace_dir: ${WORKSPACE_DIR}
+    admin_user: ${GEOSERVER_ADMIN_USER}
+    admin_password: ${GEOSERVER_ADMIN_PASSWORD}
+  Catalog:
+    active: true
+    url: https://${HOSTNAME}/twitcher/ows/proxy/catalog
```

Comment:
Since it's been changed above, shouldn't `HOSTNAME` here also have been changed to `PAVICS_FQDN_PUBLIC` ?